src/images.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import { sendRequest } from './util';
+
+export default function images(): Object {
+  const baseUrl = 'images-api.nasa.gov';
+  const endpoints = {
+    search: '/search',
+    asset: '/asset',
+    metadata: '/metadata',
+    captions: '/captions',
+  };
+
+  function validateMediaType(mediaType: string): string {
```

Comment:
The API allows for multiple search types (https://images-api.nasa.gov/search?media_type=image,audio) and this prevents them from doing that. Not sure how we would go about correctly validating this, so maybe it's simpler to just not validate (doesn't use API keys anyways).